pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -258,6 +262,16 @@
     // Name of load manager to use
     @FieldContext(dynamic = true)
     private String loadManagerClassName = "com.yahoo.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl";
+    @FieldContext(dynamic = true)
+    private String activeVersion = "";
```

Comment:
After looking at [jsemver](https://github.com/zafarkhaja/jsemver), it looks like we can go that route and git rid of `activeVersion` which will be easier operationally. Rolling back though still presents problems since bundles will get squashed onto fewer and fewer brokers which can cause issues. Also, it guarantees that most bundles will get affected twice (not once), except for the bundles on the last broker to get deployed.

To design for this, the whole feature can be disabled with a dynamic configuration value, so we will replace `activeVersion` with something like `preferLatestVersion`. But that only needs to be set during a rollback (and set again once the rollback is complete), _not_ every time a normal (forward) deployment happens.